src/components/views/messages/MImageReplyBody.tsx

```diff
@@ -43,20 +39,22 @@ export default class MImageReplyBody extends MImageBody {
             return super.render();
         }
 
-        const content = this.props.mxEvent.getContent() as IMediaEventContent;
+        const content = this.props.mxEvent.getContent<IMediaEventContent>();
 
         const contentUrl = this.getContentUrl();
-        const thumbnail = this.messageContent(contentUrl, this.getThumbUrl(), content);
+        const thumbnail = this.messageContent(contentUrl, this.getThumbUrl(), content, 44);
```

Comment:
this 44 should be a const at the top of the file with a comment of what it represents/self documenting name